prov/usnic/src/usdf.h

```diff
@@ -51,6 +51,7 @@
 #define USDF_PROV_VERSION FI_VERSION(USDF_MAJOR_VERS, USDF_MINOR_VERS)
 
 extern struct fi_provider usdf_ops;
+extern bool addr_format_str;
```

Comment:
Pretty sure you can't use this. What if more than one EP is used and only one of them is `addr_format_str`? BTW you should add that to the test cases.